Add docker build args support and image filtering (#65)
getkuby/kuby-core: docs/docs/customizing_docker_build.md

1. `copy(source, dest, from: nil)`
1. `expose(port)`
1. `cmd(command)`

> ## Docker build options

Reviewer: It would be great to mention the new `--only` flag in here somewhere 😄

1. `copy(source, dest, from: nil)`
1. `expose(port)`
1. `cmd(command)`

## Docker build options

You can pass additional build args via the `-a` (`--arg`) flag. For example, you can add a current Git commit info to a container:

```bash
bundle exec kuby build -a SOURCE_COMMIT=$COMMIT_SHA
```

> **NOTE:** You should take care of adding `ARG SOURCE_COMMIT` and `ENV SOURCE_COMMIT=$SOURCE_COMMIT` yourself (see above).

Reviewer: ```suggestion
**NOTE:** You will need to add `ARG SOURCE_COMMIT` and `ENV SOURCE_COMMIT=$SOURCE_COMMIT` to the Dockerfile yourself (see above).
```
